CromwellRefdiskManifestCreator/src/main/java/org/broadinstitute/manifestcreator/CromwellRefdiskManifestCreatorApp.java

```diff
@@ -0,0 +1,151 @@
+package org.broadinstitute.manifestcreator;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.broadinstitute.manifestcreator.exception.CRC32Exception;
+import org.broadinstitute.manifestcreator.model.ReferenceDiskManifest;
+import org.broadinstitute.manifestcreator.model.ReferenceFile;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Stack;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Stream;
+import java.util.zip.CRC32;
```

Comment:
We actually want to use `crc32c` (which google calculates and can return to us from GCS) instead of `crc32`. Despite sounding similar they're actually different hashes.

Reply:
I finally resorted to building and running it by Java 11